alibi_detect/saving/tests/models.py

```diff
@@ -25,6 +25,10 @@
 from alibi_detect.models.tensorflow import TransformerEmbedding as TransformerEmbedding_tf
 from alibi_detect.cd.pytorch import HiddenOutput as HiddenOutput_pt
 from alibi_detect.cd.tensorflow import HiddenOutput as HiddenOutput_tf
+from alibi_detect.utils.frameworks import has_keops
+if has_keops:
```

Comment:
Is this conditional purely because we skip `keops` on some platforms in CI? Should we add a comment for this? Same question for the other test module.

Reply:
Yep precisely. I'll add notes to both modules now!